.github/actions/restore-node/action.yml

```diff
@@ -122,7 +127,12 @@ runs:
     - name: git dirty check
       working-directory: ${{ inputs.path }}
       run: |-
-        if [ -n "$(git status --porcelain)" ];
+        if [ ${{ steps.endo-branch.outputs.result }} == NOPE ]; then
+          dirty=$(git status --porcelain)
+        else
+          dirty=$(git status --porcelain | egrep -ve '^ M (package\.json|yarn\.lock)$')
```

Comment:
Unfortunate, but acceptable.

Reply:
I've been thinking this over.  I'd like to pivot, and instead `git restore package.json yarn.lock` after the custom Endo's `yarn install` is done so that I can remove this special case.